index.js

```diff
@@ -82,10 +82,29 @@ module.exports = {
 								elementClass(children[i]).remove('show');
 							}
 
+							if (options.showCursor) {
+								// remove current cursor class:
+								var currentCursor = i + (mode === 'type' ? -1 : 0);
+								if (currentCursor in children) {
```

Comment:
On first step `i = 0`, and `currentCursor = i + -1`. It is just shorted than `currentCursor >= 0 && currentCursor < children.length`